style.scss

```diff
@@ -11,144 +12,264 @@ body {
+                    height: 50px;
+                }
+            }
+        }
+    }
 }
 
-.list {
-    line-height: 1.8;
-    align-self: flex-start;
-    list-style-type: none;
-    padding: 0;
-    margin: 20px;
-    font-size: 20px;
+main {
+    flex-grow: 1;
+
+    .work {
+        background-image: url("./img/man.jpg");
+        background-size: cover;
+        background-position: center;
+        height: 400px;
+
+        @media (min-width: $screen-md) {
+            height: 600px;
+        }
+    }
+
+    .text-about-us {
+        text-align: center;
+        padding-top: 100px;
+        margin: 0 300px 0 10px;
+        font-size: 1rem;
+
+        @media (min-width: $screen-sm) {
+            margin: 0 500px 0 30px;
+            font-size: 1.5rem;
+        }
+
+        @media (min-width: $screen-md) {
+            margin: 0 600px 0 50px;
+            font-size: 2rem;
+        }
+
+        @media (min-width: $screen-lg) {
+            margin: 0 600px 0 50px;
+            font-size: 2rem;
+        }
+    }
+
+    .main-title {
+        color: $primary;
+        text-align: center;
+        text-wrap: balance;
+        margin-bottom: 1rem;
+        font-size: 1rem;
+
+        @media (min-width: $screen-xs) {
+            font-size: 2rem;
+        }
+    }
+
+    .cards-container {
+        display: flex;
+        flex-direction: column;
+
+        @media (min-width: $screen-xs) {
+            flex-direction: row;
+            flex-wrap: wrap;
+        }
+    }
+
+    .card {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 1rem;
+        gap: 1rem;
+        background-color: $light;
+        color: $dark;
+
+        @media (min-width: $screen-xs) {
+            width: 50%;
+        }
+
+        @media (min-width: $screen-md) {
+            width: 33.3333333%;
+        }
+
+        &:nth-of-type(odd) {
+            background-color: $secondary-light;
+            color: $light;
+
+            @media (min-width: $screen-xs) {
+                background-color: $light;
+                color: $dark;
+            }
+
+            @media (min-width: $screen-md) {
+                background-color: $secondary-light;
+                color: $light;
+            }
+        }
+
+        &:nth-of-type(1),
+        &:nth-of-type(4),
+        &:nth-of-type(5) {
+            @media (min-width: $screen-xs) and (max-width: ($screen-md - 1)) {
+                background-color: $secondary-light;
+                color: $light;
+            }
+        }
+
+        .icon {
+            width: 50px;
+            height: 50px;
+        }
+
+        .title {
+            text-align: center;
+            font-weight: 400;
+            margin: 0;
+            font-size: 0.8rem;
+
+            @media (min-width: $screen-xs) {
+                font-size: 1rem;
+            }
+        }
+    }
 }
 
-.footer {
-    background-color: $primary-lighter;
+footer {
+    background-color: $primary-light;
```

Comment:
As I mentioned, please always (!) test your changes. The logo image is overstretched, because one rule is absent here. Please add `align-items: center` to footer and see how it's affected to the image height behavior.